src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js

```diff
@@ -96,7 +96,7 @@ const NewSSHKeyForm = ({ setStepValidated }) => {
     return regex.test(ssh);
   };
   return (
-    <>
+    <FormGroup isStack>
```

Comment:
Is it a good idea to nest FormGroups?

Reply:
with this we gain the wanted padding between form inputs with no  hardcoding css, I haven't seen any warnings or errors 🤔

Reply:
:shrug: ok, it just looked weird to me, but if it works, then it works :)